graphrag_sdk/models/litellm.py

```
@@ -232,14 +233,16 @@ def send_message(self, message: str, output_method: OutputMethod = OutputMethod.
             response = completion(
                 model=self._model.model,
                 messages=self._chat_history,
-                **generation_config
+                **generation_config,
             )
         except Exception as e:
-            raise ValueError(f"Error during completion request, please check the credentials - {e}")
+            raise ValueError(
+                f"Error during completion request, please check the credentials - {e}"
+            )
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)